_includes/tutorials/aggregating-count/confluent/markup/dev/ccloud-run-produce.adoc

```diff
@@ -0,0 +1,21 @@
+In a new terminal, run:
+
++++++
+<pre class="snippet"><code class="bash">{% include_raw tutorials/aggregating-count/confluent/code/tutorial-steps/dev/ccloud-produce-events.sh %}</code></pre>
++++++
+
+You will be prompted for the Confluent Cloud Schema Registry credentials as shown below, which you can find in the `configuration/ccloud.properties` configuration file.
```

Comment:
Is there a way for an `adoc` to reference another text file so that this can be templated this out (since it's used in so many `ccloud` KTs)

Reply:
@ybyzek Are you suggesting something like I committed here: https://github.com/confluentinc/kafka-tutorials/pull/810/commits/b76fa166166db31918e5ba371c3e1346a52a9cd3 ?

Reply:
🙌 Holla!  Totes, assuming it renders.  And as a next step, update the other tutorials with the same `include` statement?

Reply:
Yup, the only one that can't be substituted without reworking the flow of the tutorial is `_includes/tutorials/console-consumer-producer-basic/confluent/markup/dev/consume-topic-key-value.adoc`. I updated the other uses in this commit https://github.com/confluentinc/kafka-tutorials/pull/810/commits/2cafbaf27c12a0f954ad11d1fc4d3858d765855b